### PR TITLE
Refine unified schema docs

### PR DESCRIPTION
**Pull request by lpsinger**
- Clarify the relationships between JavaScript, JSON, and JSON Schema.
- Add quick reference to JSON Schema fields.
- General writing style.

### CODE REVIEW

File: app/routes/docs.notices.schema.mdx [hunk 1]
```diff
@@ -12,47 +12,63 @@ export function loader() {
   return feature('SCHEMA') || new Response(null, { status: 404 })
 }
 
-# GCN Unified Schema
+# Unified Schema
 
-Introducing the Unified Schema, a standardized framework that simplifies the creation of new GCN Notice types across various missions, observatories, and instruments, incorporating both common and mission-specific properties. We employ JavaScript Object Notation (JSON) for these Notices, a versatile internet data format with minimal ties to JavaScript. JSON offers flexibility in data representation with few structural constraints. Meanwhile, JSON Schema serves as the standard for specifying the structure, data types, and validation rules for JSON documents.
+The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema.
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:17]:
```suggestion
The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema).
```

File: app/routes/docs.notices.schema.mdx [hunk 2]
```diff
@@ -12,47 +12,63 @@ export function loader() {
   return feature('SCHEMA') || new Response(null, { status: 404 })
 }
 
-# GCN Unified Schema
+# Unified Schema
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:15]: Unified looks general, what about "Unified Schema for GCN"?

Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:15]: It's on the GCN web site, isn't that implied?

File: app/routes/docs.notices.schema.mdx (hunk 1 above)
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:17]:
```suggestion
The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema). 
```

File: app/routes/docs.notices.schema.mdx [hunk 3]
```diff
@@ -12,47 +12,63 @@ export function loader() {
...
+The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema.
 
-The process of crafting each mission schema involves the utilization of a set of core JSON schemas, forming the foundational building blocks for generating JSON-format Notices. This approach allows the flexibility of inclusion of mission-specific parameters while maximizing the use of the core schema. As a result, it ensures a consistent unit system for common properties across all missions.
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:19]:
```suggestion
The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.) The construction of mission-schema involves the utilization of a set of core JSON schemas, forming the foundational building blocks for generating JSON-format Notices. This approach allows for the flexibility of including mission-specific parameters while maximizing the use of the core schema.
```
felt there should be some intro over core schema and it's maximal use for user.

Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:19]: Yes, but that repeats the text in the paragraph that starts "Crafting a schema for a new notice type involves..."

File: app/routes/docs.notices.schema.mdx [hunk 4]
```diff
@@ -12,47 +12,63 @@ export function loader() {
...
 
-Your new schema definition files should follow the naming convention <code><i>SchemaName</i>.schema.json</code>.
-Examples corresponding to these schemas should follow the format, <code><i>SchemaName[.OptionalSpecification]</i>.example.json</code>. Both the <i>SchemaName</i> and <i>OptionalSpecification</i> example should be written in CamelCase formatting (ex., YourSchemaName, thisIsAlsoValid, etc.). This naming convention allows our [schema-browser](/docs/schema/stable/gcn) to generate pages based directly on the files in the latest tag.
+We encourage you to add files containing example data. The naming convention for example files is <code>gcn/notices/<i>mission</i>/<i>SchemaName</i>.<i>OptionalDescription</i>.example.json</code>. We will automatically validate your example files against your schema.
 
-## Sample Code
+## Definine Your Schema
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:38]:
```suggestion
## Define Your Schema
```

File: app/routes/docs.notices.schema.mdx [hunk 5]
```diff
@@ -65,8 +81,13 @@ This allows our validator script to identify which schema examples to compare. M
...
         {
           $schema:
-            'https://gcn.nasa.gov/schema/main/gcn/notices/<mission>/<schema name>.schema.json',
-          alert_datetime: 'yyyy-mm-ddThh:mm:ssZ',
+            'https://gcn.nasa.gov/schema/main/gcn/notices/mission/Sample.schema.json',
+          alert_datetime: '2023-09-28T01:40:30Z',
+          ra: 197.44871198,
+          dec: -23.38397612,
+          example_field_1:
+            'The ultimate answer to life, the universe, and everything',
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:89]: 'The ultimate answer to life, the universe, and everything' 
Sure, ROFL!

Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:89]: It's a reference to [The Hitchhiker's Guide to the Galaxy](https://en.wikipedia.org/wiki/The_Hitchhiker%27s_Guide_to_the_Galaxy), as is the number 42.

Comment (Vidushi-GitHub) [reply app/routes/docs.notices.schema.mdx:89]: 42 number!

File: app/routes/docs.notices.schema.mdx [hunk 6]
```diff
@@ -75,46 +96,64 @@ This allows our validator script to identify which schema examples to compare. M
...
+
+Declares that this schema describes a compound data type. (Leave as is.)
+
+#### `unevaluatedProperties`
+
+Declares that fields other than those listed above or inherited from the listed core schema are not allowed. (Leave as is.)
+
+#### `title`
+
+A short, human-friendly name for your schema: a short phrase.
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:119]:
```suggestion
A short, human-friendly name for your schema, typically a short phrase.
```

File: app/routes/docs.notices.schema.mdx [hunk 7]
```diff
@@ -75,46 +96,64 @@ This allows our validator script to identify which schema examples to compare. M
   </Tab>
 </Tabs>
 
-## Defining Schema
+### Quick Reference
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:99]: Good idea!

Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:99]:
```suggestion
### Quick Schema Reference
```

File: app/routes/docs.notices.schema.mdx [hunk 8]
```diff
@@ -75,46 +96,64 @@ This allows our validator script to identify which schema examples to compare. M
...
+
+List of core schema to include fields from. [Browse the core schema](/docs/schema/stable/gcn/notices/core) or [view their source on GitHub](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/core) for options.
+
+#### `properties`
+
+Optional mapping of custom, mission-specific fields. Field names should use [snake_case](https://en.wikipedia.org/wiki/Snake_case). Each entry should at least have a description and a type. See the [list of available data types](https://json-schema.org/understanding-json-schema/reference/type.html).
+
+## Extended Example
+
+We recommend exploring our [schema browser](/docs/schema/stable/gcn/notices) and -GitHub repository](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices) for more examples. Here is an extended example from Swift/BAT-GUANO:
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:135]:
```suggestion
We recommend exploring our [schema browser](/docs/schema/stable/gcn/notices) and [GitHub repository](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices) for more examples. Here is an extended example from Swift/BAT-GUANO:
```

File: app/routes/docs.notices.schema.mdx (hunk 7 above)
Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:99]: Why?

File: app/routes/docs.notices.schema.mdx (hunk 2 above)
Comment (Vidushi-GitHub) [reply app/routes/docs.notices.schema.mdx:15]: Ya that also I felt, but second thought that was self-consistent.

File: app/routes/docs.notices.schema.mdx (hunk 3 above)
Comment (Vidushi-GitHub) [reply app/routes/docs.notices.schema.mdx:19]: You are right, sorry. Then should add a single comment "we encourage max use of core-schema wherever possible."

Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:19]: Not in this paragraph, though. This paragraph is about JSON and JSON Schema.

File: app/routes/docs.notices.schema.mdx (hunk 7 above)
Comment (Vidushi-GitHub) [reply app/routes/docs.notices.schema.mdx:99]: Wanted to be as explicit as possible

File: app/routes/docs.notices.schema.mdx (hunk 3 above)
Comment (Vidushi-GitHub) [reply app/routes/docs.notices.schema.mdx:19]: Sure, not in this paragraph, I meant for "Crafting a schema for a new notice type involves..." para end.

File: app/routes/docs.notices.schema.mdx (hunk 7 above)
Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx:99]: How about "JSON Schema Quick Reference"?

File: app/routes/docs.notices.schema.mdx [hunk 9]
```diff
@@ -12,47 +12,63 @@ export function loader() {
...
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
 
-## Fork the GitHub Repository
+The definition of the GCN Unified Schema is itself expressed in JSON format using the [JSON Schema](https://json-schema.org) standard. JSON Schema makes it possible for us to provide automatic, interactive documentation of the schema in our [Schema Browser](/docs/schema).
+
+Crafting a schema for a new notice type involves selecting one or more of the predefined [core schema](/docs/schema/stable/gcn/notices/core) and adding your own optional mission-specific fields. This approach allows the flexibility of inclusion of mission-specific parameters while ensuring consistency where possible.
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:23]:
```suggestion
Crafting a schema for a new notice type involves selecting one or more of the predefined [core schema](/docs/schema/stable/gcn/notices/core) and adding your own optional mission-specific fields. This approach allows the flexibility of inclusion of mission-specific parameters while ensuring consistency where possible, thus, we encourage producers to utilize the core schema as much as possible.
```
I am emphasizing as I found Josh mentioned that if he can use specific field in fermi/gbm for defining TJD time as well.

**Comment by Vidushi-GitHub** [comment]
Looks good! Is there a way to see website version, other than forking your branch, to see final look?

File: app/routes/docs.notices.schema.mdx [hunk 10]
```diff
@@ -12,47 +12,62 @@ export function loader() {
...
 
-The process of crafting each mission schema involves the utilization of a set of core JSON schemas, forming the foundational building blocks for generating JSON-format Notices. This approach allows the flexibility of inclusion of mission-specific parameters while maximizing the use of the core schema. As a result, it ensures a consistent unit system for common properties across all missions.
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
 
-## Fork the GitHub Repository
+The definition of the GCN Unified Schema is itself expressed in JSON format using the [JSON Schema](https://json-schema.org) standard. JSON Schema makes it possible for us to provide automatic, interactive documentation of the schema in our [Schema Browser](/docs/schema).
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:21]: Needed to replaced with final schema browser link, I added when that was in production.

Comment (lpsinger) [reply app/routes/docs.notices.schema.mdx]: No, a path without a URL origin is preferable.

**Comment by lpsinger** [comment]
Does this need anything else?

**Comment by Vidushi-GitHub** [comment]
content wise done! I am just figuring out to clone your branch, to see the website look.

File: app/routes/docs.notices.schema.mdx [hunk 11]
```diff
@@ -12,47 +12,62 @@ export function loader() {
...
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
 
-## Fork the GitHub Repository
+The definition of the GCN Unified Schema is itself expressed in JSON format using the [JSON Schema](https://json-schema.org) standard. JSON Schema makes it possible for us to provide automatic, interactive documentation of the schema in our [Schema Browser](/docs/schema/stable/gcn).
+
+Crafting a schema for a new notice type involves selecting one or more of the predefined [core schema](/docs/schema/stable/gcn/notices/core) and adding your own optional mission-specific fields. This approach allows the flexibility of inclusion of mission-specific parameters while ensuring consistency where possible. We encourage producers to utilize the core schema as much as possible.
+
+As you are develping your schema, don't hesitate to [contact us](/contact) with any questions!
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:25]:
```suggestion
As you are developing your schema, don't hesitate to [contact us](/contact) with any questions!
```

File: app/routes/docs.notices.schema.mdx [hunk 12]
```diff
@@ -12,47 +12,62 @@ export function loader() {
   return feature('SCHEMA') || new Response(null, { status: 404 })
 }
 
-# GCN Unified Schema
+# Unified Schema
 
-Introducing the Unified Schema, a standardized framework that simplifies the creation of new GCN Notice types across various missions, observatories, and instruments, incorporating both common and mission-specific properties. We employ JavaScript Object Notation (JSON) for these Notices, a versatile internet data format with minimal ties to JavaScript. JSON offers flexibility in data representation with few structural constraints. Meanwhile, JSON Schema serves as the standard for specifying the structure, data types, and validation rules for JSON documents.
+The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are [joining GCN as a new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema).
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:17]:
```suggestion
The GCN Unified Schema is a framework for defining GCN Notice types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are joining GCN as a [new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema).
```

Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:17]:
```suggestion
The GCN Unified Schema is a framework for defining GCN Notices types using a common format, standardized field names, uniform data types, and consistent physical units across multiple missions. If you are joining GCN as a [new notice producer](/docs/notices/producers), then you can contribute your own notice types to the Unified Schema present at [GitHub](https://github.com/nasa-gcn/gcn-schema).
```

File: app/routes/docs.notices.schema.mdx [hunk 13]
```diff
@@ -12,47 +12,62 @@ export function loader() {
...
 
-The process of crafting each mission schema involves the utilization of a set of core JSON schemas, forming the foundational building blocks for generating JSON-format Notices. This approach allows the flexibility of inclusion of mission-specific parameters while maximizing the use of the core schema. As a result, it ensures a consistent unit system for common properties across all missions.
+The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. (Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.)
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:19]:
```suggestion
The over-the-wire format for GCN Notices in the Unified Schema is [JavaScript Object Notation (JSON)](https://en.wikipedia.org/wiki/JSON), an Internet standard for encoding arbitrary data as human-readable text. Despite having "JavaScript" in its name, JSON is ubiquitous, and many programming languages have standard library support for JSON.
```
It looks like a single line paragraph with brackets. Other option is merge with next paragraph.

File: app/routes/docs.notices.schema.mdx [hunk 14]
```diff
@@ -12,47 +12,62 @@ export function loader() {
...
+
+Crafting a schema for a new notice type involves selecting one or more of the predefined [core schema](/docs/schema/stable/gcn/notices/core) and adding your own optional mission-specific fields. This approach allows the flexibility of inclusion of mission-specific parameters while ensuring consistency where possible. We encourage producers to utilize the core schema as much as possible.
+
+As you are develping your schema, don't hesitate to [contact us](/contact) with any questions!
 
-To begin, initiate by forking your own copy of the [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn-schema) repository on GitHub. Next, proceed to clone a local version and follow the project configuration instructions available at [GitHub primer](/docs/contributing/github).
+## Fork the GitHub Repository
 
-## Setting Up Your Repository
+To get started, fork the [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn-schema) repository on GitHub and check it out on your computer. If you are new to GitHub, see our [GitHub primer](/docs/contributing/github).
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.schema.mdx:29]:
```suggestion
To get started, fork the [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn-schema) repository on GitHub and check it out on your computer. If you are new to GitHub, refer to our [GitHub primer](/docs/contributing/github).
```
Optional comment!

**Review by Vidushi-GitHub — APPROVED** [review]

**Review by Vidushi-GitHub — APPROVED** [review]

**Pull request merged**